TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/fsm/StateMachine.java

```diff
@@ -32,42 +108,50 @@ public StateMachine addState(State state) {
      * @return The state machine.
      */
     public StateMachine deleteState(State state) {
```

Comment:
wwhat- why? Just delete the class that declares the state

Reply:
Fixed in [`b177523`](https://github.com/rh-robotics/19922-CSRC/pull/42/commits/b17752345df9febcdbc5c42d3553eb8f946ae00f).